### ISSUE TITLE
Cannot add IP manually, entry form blocked behind "Searching" overlay

### ISSUE THREAD

**Issue by Nindaleth** [pascalopitz/unoffical-sonos-controller-for-linux#142]
Hello, I'd love to try the Unofficial Sonos Controller, however I'm hitting an issue right at the beginning.

**Environment**

* I'm on Fedora 34 and have installed the official RPM 0.2.5 from the Releases page.
* Here's the list of my opened ports as listed by the active firewalld:
`ports: 1025-65535/udp 1025-65535/tcp`

**Issue**

* getting the eternal "Searching for your Sonos system..." even though the speaker is on the same network (I can see its IP from the Android app) and my ports shouldn't be blocked by the firewall
* tried the `Developer -> Add IP manually` but I only get this, I'd probably call it pop-under (see the faint window under the overlay)
* doesn't seem like I can type into the underlying window/form
![add_ip_manually](https://user-images.githubusercontent.com/6383928/121814320-a44f5c80-cc70-11eb-8e59-82b68d900f3e.png)

Is there anything else I can do about this?

**Comment by pascalopitz** [comment]
Odd. Is that on a snap install? Can you try the AppImage and check if that's acting the same?

**Comment by Nindaleth** [comment]
I've tried the almost identical Fedora 34 + RPM environment that I have on my work computer within the same network - the speaker is found immediately.

The difference is that on my personal computer I'm running Mullvad (a VPN client) - there's an exception in place for local network addresses because I need to access my local NAS, phone etc. I've tried the following:
* disconnecting the VPN and restarting Sonos Controller - fixes the issue
* the VPN supports starting apps outside of the VPN which was verified to work as intended with other apps, doesn't help here though

I can only guess; from the observed behaviour I'd say that it's the additional network interface created by the VPN client that's causing trouble in some way.
